src/extension.ts

```diff
@@ -36,6 +36,13 @@ export class EdgeConfigurationProvider implements vscode.DebugConfigurationProvi
      * Try to add all missing attributes to the debug configuration being launched.
      */
     async resolveDebugConfiguration(folder: vscode.WorkspaceFolder | undefined, config: vscode.DebugConfiguration, token?: vscode.CancellationToken): Promise<vscode.DebugConfiguration> {
+        if (!isEdgeDebuggingSupported()) {
+            const errorMessage = localize('edge.debug.error.versionNotSupported', "Edge debugging is not supported for your version of Windows. Please update to the latest version.");
```

Comment:
In case we have a bug in detecting the version of windows, I'd add the version of windows we detected to this message, and the version of windows we are looking for so the user can make sense of what is happening.
Also, this message doesn't make sense if the user is on Linux, etc...

Reply:
We should actually say: "Edge doesn't support debugging from VS Code".
Edge does support debugging through F12...

Reply:
Updated the PR to use better wording.